src/plugins/open/open.c

```diff
@@ -170,7 +170,7 @@ static
 struct ipmi_rs *
 ipmi_openipmi_send_cmd(struct ipmi_intf *intf, struct ipmi_rq *req)
 {
-	struct ipmi_recv recv;
+	struct ipmi_recv recv = {};
```

Comment:
Good catch.
I see the same problem in `ipmievd.c:openipmi_read()`, would be nice to fix it there too.